Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -26,11 +26,12 @@
 
 WI.InlineSwatch = class InlineSwatch extends WI.Object
 {
-    constructor(type, value, readOnly)
+    constructor(type, value, {readOnly, preventChangingColorFormats} = {})
```

Comment:
NIT: Should we include `value` in there too?  the usage further down makes it seem like it's not required.